packages/ordered-list/src/OrderedList/OrderedList.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { ListItem } from '../ListItem';
+
+import { OrderedList } from './OrderedList';
+
+const renderOL = () => {
+  return render(
+    <OrderedList>
+      <ListItem title="Title" description="Description" />
+      <ListItem title="Title" description="Description" />
+      <ListItem title="Title" description="Description" />
+    </OrderedList>,
+  );
+};
+
+describe('packages/typography/ordered-list', () => {
+  describe('OrderedList', () => {
+    test('renders an ordered list', () => {
+      renderOL();
+
+      const ol = screen.getByRole('list', { name: 'OrderedList' });
```

Comment:
Looks like this is failing because [list role is named by author](https://www.w3.org/TR/wai-aria-1.2/#list) and won't have this name.
```suggestion
      const ol = screen.getByRole('list');
```